src/processors/typescript.ts

```
@@ -111,6 +111,14 @@ export class TypeScriptFileProcessor implements FileProcessor {
       filesList.push(TypeScriptFileProcessor.getEntryPointImport(file));
     }
 
+    const includes = this.includesFromNearestTsconfigFile(file);
+    if (includes.length) {
+      // only include non-globbed includes
+      // TODO: support globbed includes
```

Agree it doesn't need to be implemented immediately, but can we throw here instead of silently ignoring these?
Also, is there anything in the TS tooling that could potentially resolve these for us?

---

Also something just crossed my mind - with extended tsconfigs, do we need to resolve the whole tree of includes? I haven't checked the docs but my gut feel would be that includes and excludes stack through tsconfig extensions, which would be one more reason to use the original code to resolve.

---

> It’s worth noting that [files](https://www.typescriptlang.org/tsconfig#files), [include](https://www.typescriptlang.org/tsconfig#include) and exclude from the inheriting config file overwrite those from the base config file, and that circularity between configuration files is not allowed.

Apparently not! https://www.typescriptlang.org/tsconfig#extends